custom_components/solarfocus/sensor.py

```diff
@@ -564,6 +566,27 @@ def native_value(self):
         device_class=SensorDeviceClass.ENUM,
         options=list(range(0, 2)),
     ),
+    SensorEntityDescription(
+        key="pellet_usage_last_fill",
+        native_unit_of_measurement=MASS_KILOGRAMS,
+        icon="mdi:gradient-vertical",
+        device_class=SensorDeviceClass.WEIGHT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="pellet_usage_total",
+        native_unit_of_measurement=MASS_KILOGRAMS,
+        icon="mdi:alpha-t-box",
+        device_class=SensorDeviceClass.WEIGHT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="heat_energy_total",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        icon="mdi:fire-circle",
```

Comment:
to be consistent I'd use "mdi:meter-gas" for thermal energy cf

https://github.com/LavermanJJ/home-assistant-solarfocus/blob/6140137d6f7c6da526efae0d66493185594d042c/custom_components/solarfocus/sensor.py#L371-L377